WalletWasabi.Tests/UnitTests/BlockchainAnalysis/PubKeyReuseAnonScoreTests.cs

```diff
@@ -35,6 +35,45 @@ public void AddressReusePunishment()
 			Assert.True(tx.WalletOutputs.First().HdPubKey.AnonymitySet < 30);
 		}
 
+		[Fact]
+		public void AddressReusePunishmentProcessTwice()
+		{
+			// If there's reuse in input and output side, then output side didn't gain, nor lose anonymity.
```

Comment:
Is this comment relevant here too? I can see it several times in the test file which is somewhat suspicious.

Reply:
No, please remove it guys, good catch.

Reply:
Done